_data/simple-icons.json

```
@@ -17941,6 +17941,12 @@
             "hex": "009CFB",
             "source": "https://www.zingat.com"
         },
+        {
+            "title": "Zip Pay",
+            "hex": "1A0826",
+            "source": "https://zip.co",
```

Let's use the icon provided in the press kit linked below as the source, make sure to verify against the current or update the icon.

```suggestion
            "source": "https://zip.co/nz/marketing/the-logo/",
```